control_panel_api/tests/permissions/test_s3bucket_permissions.py

```diff
@@ -18,28 +19,32 @@ class S3BucketPermissionsTest(APITestCase):
 
     def setUp(self):
         super().setUp()
-        # Create users
-        self.superuser = mommy.make(
-            'control_panel_api.User', is_superuser=True)
-        self.normal_user = mommy.make(
-            'control_panel_api.User', is_superuser=False)
-        # Create some S3 buckets
-        self.s3bucket_1 = mommy.make(
-            "control_panel_api.S3Bucket", name="test-bucket-1")
-        self.s3bucket_2 = mommy.make(
-            "control_panel_api.S3Bucket", name="test-bucket-2")
+
+        self.superuser = mommy.make('control_panel_api.User',
+                                    is_superuser=True)
```

Comment:
Just a pet peeve, as this indentation style is ok according to PEP8, but I think indenting like this is better:
```python
self.superuser = mommy.make(
    'control_panel_api.User',
    is_superuser=True)
```
It keeps indentation levels regular throughout the code, instead of having indentation depend on the length of the variable name. Also, if you change the length of the variable name for whatever reason, you won't have to re-indent the following line(s).

Reply:
That indentation is coming straight from my PEP8 plugin BTW.

Reply:
Re-read your comment, I see. I can change these 👍

Reply:
(I just tried to now think too much about it as it was coming from the PEP8 plugin and I didn't want to question the "word" :P  )

Reply:
FYI: https://github.com/ministryofjustice/analytics-platform-control-panel/pull/97/commits/91de3d8b29bdb634723d93beeca78c10f74e0ef6